languagetool-language-modules/no/src/main/java/org/languagetool/tagging/no/NorwegianTagger.java

```diff
@@ -0,0 +1,12 @@
+package org.languagetool.tagging.no;
+
+import org.languagetool.tagging.BaseTagger;
+
+import java.util.Locale;
+
+public class NorwegianTagger extends BaseTagger {
+  public NorwegianTagger() {
+//    super("/no/norwegian.dict", new Locale("no"));
+    super("/no/nb_NO.dict", new Locale("nb", "NO"));
```

Comment:
Shouldn't it be `no` here?

Reply:
we could use `norwegian.dict` (the one i steal from https://github.com/EirikBirkeland/languagetool/tree/master/languagetool-language-modules/no/src/main/resources/org/languagetool/resource/no), not from URL requested in the JIRA (newer & better), another the point is we want to make `Norwegian Bokmål` (more popular) default lookup dictionary.